docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -3,87 +3,147 @@
+. Create an API key from your AI provider to authenticate requests from the AI Assistant. You'll use this in the next step. Refer to your provider's documentation for information on generating API keys:
 +
 * https://platform.openai.com/docs/api-reference[OpenAI]
 * https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
 
-. Update {kib}'s configuration settings to include the Observability AI Assistant feature flag and additional information about your AI set up.
-+
-.. Navigate to the {kib} configuration settings according to your deployment:
-+
-** *Self-managed (on-premises) deployments*: Add the configuration to the `kibana.yml` file, which is used to {kibana-ref}/settings.html[configure {kib}].
-** *{ecloud} deployments*: Use the YAML editor in the {ecloud} console to add the configuration to {cloud}/ec-manage-kibana-settings.html[{kib} user settings].
-+
-.. Add one of the following configurations that corresponds to the AI service you're using:
-+
-** *OpenAI*:
-+
-In the following example, `xpack.observability.aiAssistant.provider.openAI.apiKey` is the API key you generated in the first step.
-+
-[source,yaml]
-----
-xpack.observability.aiAssistant.enabled: true
-xpack.observability.aiAssistant.provider.openAI.apiKey: <insert API key>
-xpack.observability.aiAssistant.provider.openAI.model: <insert model name, e.g. gpt-4>
-----
-+
-** *Azure OpenAI Service*:
-+
-In the following example, `xpack.observability.aiAssistant.provider.azureOpenAI.apiKey` is the API key you generated in the first step, and `xpack.observability.aiAssistant.provider.azureOpenAI.resourceName` is the name of the Azure OpenAI resource used to deploy your model. 
-+
-[source,yaml]
+. From *{stack-manage-app}* -> *{connectors-ui}* in {kib}, create a {kibana-ref}/gen-ai-action-type.html[Generative AI connector]. 
+. Authenticate communication between {observability} and the AI provider by providing the following information:
+.. Enter the AI provider's API endpoint URL in the *URL* field.
+.. Enter the API key you created in the previous step in the *API Key* field. 
+
+[discrete]
+[[obs-ai-add-data]]
+== Add data to the AI Assistant knowledge base
+
+The AI Assistant uses {ml-docs}/ml-nlp-elser.html[ELSER], Elastic's semantic search engine, to recall data from its internal knowledge base index to create retrieval augmented generation (RAG) responses. Adding data such as Runbooks, Github issues, internal documentation, and Slack messages to the knowledge base index gives the AI Assistant context to provide more specific assistance. 
+
+NOTE: Your AI provider may collect telemetry when using the AI Assistant. Contact your AI provider for information on how data is collected.
+
+To add data to the AI Assistant knowledge base index:
```

Comment:
Should we also add that it's possible to have the assistant add information to the kb itself by requesting it to remember information in a conversation? @dgieselaar

Reply:
Yes, good point. I think it's an easy way for users to add things to the knowledge base without going through Elasticsearch APIs

Reply:
@CoenWarmer @dgieselaar  What's the process for getting the assistant to remember info from a conversation?

Reply:
It will sometimes do it by itself once it has figured out the user has told them something new, but if the user is very explicit about it (e.g. "Remember this for next time") it will also create a summary of what it is supposed to remember and store it in Elasticsearch.